storj/node_test.go

```diff
@@ -146,13 +146,13 @@ func TestNodeID_String_Version(t *testing.T) {
 
 func TestNodeID_Compare(t *testing.T) {
 	var a storj.NodeID
-	require.Equal(t, 0, a.Compare(a))
+	require.Equal(t, 0, a.Compare(a)) //nolint: gocritic
 
 	for k := 0; k < len(storj.NodeID{}); k++ {
 		var a, b storj.NodeID
 		a[k], b[k] = 1, 2
-		require.Equal(t, 0, a.Compare(a))
-		require.Equal(t, 0, b.Compare(b))
+		require.Equal(t, 0, a.Compare(a)) //nolint: gocritic
+		require.Equal(t, 0, b.Compare(b)) //nolint: gocritic
```

Comment:
Is there a way to specific a specific reason instead of disabling the whole linter?